openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -547,4 +547,8 @@ jQuery(function () {
             .then(module => module.initBreadcrumbSelect(crumbs));
     }
 
+    if (document.getElementById('ia-third-party-logins')) {
```

Comment:
Can we do something like this so we only do the query once and pass it in?
https://github.com/internetarchive/openlibrary/pull/8374/files#diff-587fb865c85bb3325504c795a40db189d2e702d42d9d160304607e027c142ebd

Reply:
```js
const thirdPartyLoginsIframe = document.getElementById('ia-third-party-logins');
```
Does the name `thirdPartyLoginsIframe` look good?